src/extensibility/ExtensionManager.js

```diff
@@ -398,6 +408,60 @@ define(function (require, exports, module) {
         }
         return result.promise();
     }
+    
+    /**
+     * Disables the installed extension with the given id.
+     * 
+     * @param {string} id The id of the extension to disable.
+     * @return {$.Promise} A promise that's resolved when the extenion is disabled or
+     *      rejected with an error that prevented the disabling.
+     */
+    function disable(id) {
+        var result = new $.Deferred(),
+            extension = extensions[id];
+        if (extension && extension.installInfo) {
+            Package.disable(extension.installInfo.path)
+                .done(function () {
+                    extension.installInfo.status = DISABLED;
+                    extension.installInfo.metadata.disabled = true;
+                    result.resolve();
+                    exports.trigger("statusChange", id);
+                })
+                .fail(function (err) {
+                    result.reject(err);
+                });
+        } else {
+            result.reject(StringUtils.format(Strings.EXTENSION_NOT_INSTALLED, id));
+        }
+        return result.promise();
+    }
+    
+    /**
+     * Enables the installed extension with the given id.
+     * 
+     * @param {string} id The id of the extension to enable.
+     * @return {$.Promise} A promise that's resolved when the extenion is enabled or
+     *      rejected with an error that prevented the enabling.
+     */
+    function enable(id) {
+        var result = new $.Deferred(),
```

Comment:
OK, whatever. I disagree completely and I don't understand why this was worth such a discussion. I will make the change you're asking for just to end it. Added lines count will be more than remove lines count.

P.S. I think it would be helpful if someone with the authority would document this in code style guides so that nobody would have to lose their time in a discussion like this.

P.P.S. [bikeshed](http://bikeshed.com/)